### PR TITLE
SSO Doc

### PR DESCRIPTION
**Pull request by Kakulukian**
For https://github.com/huggingface/moon-landing/issues/6822

### CODE REVIEW

File: docs/hub/security-sso.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Single sign-on (SSO)
+
+With Hugging Face Hub, you have the option to implement mandatory Single Sign-On (SSO) for members of your organization. You can activate your SSO provider for a particular email domain (e.g., @example.com). Users with an email address that matches the domain must log in on huggingface.co using your SSO provider to access your organization.
```
Comment (gary149) [inline docs/hub/security-sso.md:3]:
```suggestion
The Hugging Face Hub gives you the ability to implement mandatory Single Sign-On (SSO) for members of your organization. You can enable your SSO provider for a specific email domain (for example, @example.com). When enabled, users with an email address that matches the domain must log in to huggingface.co using your SSO provider to access your organization.
```

File: docs/hub/security-sso.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+- Your members will see a banner prompting them to authenticate through your chosen IdP to access your organization.
+
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/security-sso-prompt.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/security-sso-prompt-dark.png"/>
+</div>
+
+- If an end-user is a member of any organization outside yours, they are still required to log in via SSO as soon as the feature is enabled to access public content of your organization. However, they will be able to access other teams.
```
Comment (gary149) [inline docs/hub/security-sso.md:22]:
```suggestion
- If an end user is a member of another organization, they will still need to log in via SSO as soon as the feature is enabled to access your organization's public content. However, they will be able to access other teams.
```

File: docs/hub/security-sso.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Your members will see a banner prompting them to authenticate through your chosen IdP to access your organization.
+
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/security-sso-prompt.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/security-sso-prompt-dark.png"/>
+</div>
+
+- If an end user is a member of another organization, they will still need to log in via SSO as soon as the feature is enabled to access your organization's public content. However, they will still have access to their other organizations.
+- If you enable the role mapping feature, the admin from your organization won't be able to change their role. It will be determined by your IdP and your settings.
```
[the remark this answers is not in the document]
Comment (Kakulukian) [reply docs/hub/security-sso.md:23]: it's done manually as long as we don't have self-service yet

File: docs/hub/security-sso.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- Issuer URL
+
+## Supported SSO Partners
+You can easily integrate Hugging Face Hub with a variety of SSO providers, and we have listed a few examples below:
+
+1. Okta
+2. OneLogin
+3. Azure Active Directory (Azure AD)
+
+These are just some of the popular examples, but Hugging Face Hub can work with any OIDC-compliant SSO system.
```
Comment (Pierrci) [inline docs/hub/security-sso.md:40]:
```suggestion

You can easily integrate Hugging Face Hub with a variety of Identity Providers, such as Okta, OneLogin or Azure Active Directory (Azure AD). Hugging Face Hub can work with any OIDC-compliant Identity Provider.
```

**Review by Pierrci — COMMENTED** [review]
Alright coming back to this; I added a bunch of suggestions, mostly rewordings, it looks good! 👍

File: docs/hub/security-sso.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# Single sign-on (SSO)
+
+The Hugging Face Hub gives you the ability to implement mandatory Single Sign-On (SSO) for members of your organization. You can enable your SSO provider for a specific email domain (for example, @example.com). When enabled, users with an email address that matches the domain must log in to huggingface.co using your SSO provider to access your organization.
+
+<Tip warning={true}>
+This feature is available for Enterprise Hub. <a href="mailto:sales@huggingface.co" target="_blank">For more information, please contact our sales team</a>.
```
[the remark this answers is not in the document]
Comment (Kakulukian) [reply docs/hub/security-sso.md:6]:
```suggestion
This feature is available for Enterprise Hub. <a href="https://huggingface.co/enterprise" target="_blank">For more information, please contact our sales team</a>.
```

File: docs/hub/security-sso.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Single Sign-On (SSO)
+
+The Hugging Face Hub gives you the ability to implement mandatory Single Sign-On (SSO) for members of your organization. You can enable your SSO provider for a specific email domain (for example, @example.com). When enabled, users with an email address that matches the domain must log in to huggingface.co using your SSO provider to access your organization.
+
+<Tip warning={true}>
+This feature is available for Enterprise Hub. <a href="https://huggingface.co/enterprise" target="_blank">For more information, please contact our sales team</a>.
```
Comment (Pierrci) [inline docs/hub/security-sso.md:6]:
```suggestion
This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```

File: docs/hub/security-sso.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Single Sign-On (SSO)
+
+The Hugging Face Hub gives you the ability to implement mandatory Single Sign-On (SSO) for members of your organization. You can enable your SSO provider for a specific email domain (for example, @example.com). When enabled, users with an email address that matches the domain must log in to huggingface.co using your SSO provider to access your organization.
```
Comment (Pierrci) [inline docs/hub/security-sso.md:3]:
```suggestion
The Hugging Face Hub gives you the ability to implement mandatory Single Sign-On (SSO) for your organization.
```
(the rest is a bit redundant with the following section I think)

File: docs/hub/security.md [hunk 8]
```diff
@@ -15,6 +15,7 @@ For any other security questions, please feel free to send us an email at securi
 - [User Access Tokens](./security-tokens)
 - [Git over SSH](./security-git-ssh)
 - [Signing commits with GPG](./security-gpg)
+- [Single Sign-On (SSO)](./security-sso)
```
Comment (Pierrci) [inline docs/hub/security.md:18]: I'm wondering if it wouldn't make more sense under `Organizations`, but no strong opinion, cc @julien-c

**Review by Pierrci — APPROVED** [review]
Last nits, LGTM!

File: docs/hub/security-sso.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+To successfully enable SSO on the Hugging Face Hub for your organization, you need to set up an OIDC application in your IdP. We will provide a callback URL pointing to the `huggingface.co` domain. Once you create the application, you will need to provide us with the following information:
+
+- Client ID
+- Client Secret
+- Issuer URL
+
+## Supported Identity Providers
+
+You can easily integrate Hugging Face Hub with a variety of Identity Providers, such as Okta, OneLogin or Azure Active Directory (Azure AD). Hugging Face Hub can work with any OIDC-compliant Identity Provider.
```
Comment (julien-c) [inline docs/hub/security-sso.md:35]: can we say something about upcoming support for SAML?

File: docs/hub/security.md (hunk 8 above)
Comment (julien-c) [reply docs/hub/security.md:18]: i would keep it under security but link it from https://huggingface.co/docs/hub/organizations , https://huggingface.co/docs/hub/organizations-managing and from https://huggingface.co/docs/hub/organizations-security

**Review by julien-c — APPROVED** [review]

**Review by SBrandeis — APPROVED** [review]

File: docs/hub/security.md (hunk 8 above)
Comment (Kakulukian) [reply docs/hub/security.md:18]: I'll merge without it. Let's do that when I have the UI ready

**Review by julien-c — COMMENTED** [review]
let's remember to add a link to this page from https://huggingface.co/docs/hub/organizations-security and possibly from https://huggingface.co/docs/hub/organizations-managing

**Pull request merged**